mep/books/models.py

```diff
@@ -259,6 +314,21 @@ def format(self):
         '''format of this work if known (e.g. book or periodical)'''
         return self.work_format.name if self.work_format else ''
 
+    index_depends_on = {
```

Comment:
this could also move into the `SignalHandlers` or `Indexing` associated class, if we made one.

Reply:
Agree